contrib/avro/src/python/pants/contrib/avro/tasks/BUILD

```diff
@@ -0,0 +1,15 @@
+python_library(
+  name = 'tasks',
+  sources = globs('*.py'),
```

Comment:
`sources` is redundant here, since the argument matches the default for a python_library.

(for this and the other item, see https://www.pantsbuild.org/build_files.html#target-definitions)

Reply:
Done.